app/models/event.rb

```diff
@@ -25,13 +25,16 @@ class Event < ActiveRecord::Base
   scope :recent, -> { order('name ASC') }
   scope :live, -> { where("state = 'open' and (closes_at is null or closes_at > ?)", Time.current).order('closes_at ASC') }
 
-  validates :name, :contact_email, presence: true
+  validates :name, presence: true
   validates :slug, presence: true, uniqueness: true
-  validates :closes_at, presence: true
 
   before_validation :generate_slug
   before_save :update_closes_at_if_manually_closed
 
+  STATUSES = { open: 'open',
+               draft: 'draft',
+               closed: 'closed' }
+
```

Comment:
i think it's fine for now. this assumes the identifier for the status and the display of it will be identical, which makes sense for this app.